content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,61 @@ positional arguments:
 
 In order to track parameters and hyperparameters associated to machine learning
 experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. Parameters are defined using the the `-p`
-(`--params`) option of `dvc run`, using simple names like `epochs`,
+dependencies: _parameters_. They usually have simple names like `epochs`,
 `learning-rate`, `batch_size`, etc.
 
 In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file (or
-directory). Instead, it consists of a _parameter name_ (or key) to find inside a
-YAML 1.2, JSON, TOML, or [Python](#examples-python-parameters-file) _parameters
-file_. Multiple parameter dependencies can be specified from one or more
-parameters files.
+directory). Instead, it consists of a _parameter name_ (or key) in a
+[parameters file](#parameters-file), where the _parameter value_ should be
+found. This allows you to define [stage](/doc/command-reference/run)
+dependencies more granularly: changes to other parts of the params file will not
+affect the stage. Parameters also prevent situations where several stages share
+a common dependency (e.g. a config file), and any change in it invalidates all
+these stages, causing unnecessary re-executions upon `dvc repro`.
+
+To start tracking parameters from the default `params.yaml` file, list them
```

Comment:
I would put this paragraph and example above the previous long explanation. Especially this short example.

Reply:
To start tracking parameters ... (let's not specify in this way that they are from params.yaml - you put too much focus on a spec detail). I would split this:

To start tracking parameters, specify under the `params` field of `dvc.yaml` stages (manually or with the the
`-p`/`--params` option of `dvc run`):

```
example
```

all other long explanations here ....

Reply:
Good idea, I started with the example first now.

Keep in mind that a lot of this content will be extracted to the basic concepts section as soon as I have time to take that project again so let's not focus too much on the exact order of ideas here? But I did change it like you suggested 🙂